modular_skyrat/modules/mapping/code/areas/centcom.dm

```diff
@@ -33,3 +33,11 @@
 
 /area/centcom/interlink
 	name = "The Interlink"
+
+/*
+creating an event subtype of the interlink area, so it follows the same logic and can be renamed
+without renaming the interlink itself
+*/
+
+/area/centcom/interlink/event
+	name = "Event Venue"
```

Comment:
I feel very mixed on this and don't know if we really need it.

IIRC build tools allows admins to create areas(?). If not, then that's something we can push upstream.

Reply:
I think it allows you to create areas, but not ones that are prepowered and no atmos like the interlink is

Reply:
Either way, not sure how to use that with maps made through strongdmm and loaded as a map template in game

Reply:
> Either way, not sure how to use that with maps made through strongdmm and loaded as a map template in game

This can be done through Test Merges then. There is no reason to include a prepared area "template" for usage.

Ideally, and honestly, big events (like weddings) are planned well in advanced and, even building it in game takes a long time. I'd strongly recommend individuals looking to make wedding events to contact Golden or Cobalt about getting stuff TM'd.

Reply:
> I think it allows you to create areas, but not ones that are prepowered and no atmos like the interlink is

you can varedit areas to do this